src/pages/workspace/WorkspacesListPage.tsx

```diff
@@ -107,30 +106,27 @@ function dismissWorkspaceError(policyID, pendingAction) {
     throw new Error('Not implemented');
 }
 
-// TODO: Rewrite this component to TS according to existing migration on the main branch
-function WorkspacesListPage({policies, allPolicyMembers, reimbursementAccount, reports}) {
+function WorkspacesListPage({policies, allPolicyMembers, reimbursementAccount, reports}: WorkspaceListPageProps) {
     const theme = useTheme();
     const styles = useThemeStyles();
     const {translate} = useLocalize();
     const {isOffline} = useNetwork();
     const {isSmallScreenWidth} = useWindowDimensions();
 
     const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false);
-    const [policyIDToDelete, setPolicyIDToDelete] = useState(null);
-    const [policyNameToDelete, setPolicyNameToDelete] = useState(null);
+    const [policyIDToDelete, setPolicyIDToDelete] = useState('');
+    const [policyNameToDelete, setPolicyNameToDelete] = useState('');
```

Comment:
I think the following solution might be safer here. In `confirmDeleteAndHideModal` function we will need to check if the policy to delete is set.
```suggestion
    const [policyIDToDelete, setPolicyIDToDelete] = useState<string>();
    const [policyNameToDelete, setPolicyNameToDelete] = useState<string>();
```